libs/shared/src/containers/Login/Login.stories.tsx

```diff
@@ -14,6 +13,6 @@ export default {
 };
 
 export const Login = () => {
-  const account = AccountMap.useData().map[NormalizeAddress("445758e37f47b44e05e74ee4799f3469de62a2cb")];
+  const account = AccountMap.useAsArray()[0];
```

Comment:
both lines should have worked, but only one did.  Strange, but not high-enough priority to figure out what's happened